streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -323,18 +323,17 @@ void setConsumer(final Consumer<byte[], byte[]> consumer) {
      * @throws TaskMigratedException if another thread wrote to the changelog topic that is currently restored
      */
     boolean updateNewAndRestoringTasks() {
-        final Set<TopicPartition> resumed = active.initializeNewTasks();
+        active.initializeNewTasks();
```

Comment:
I think we can cleanup `initializeNewTasks()` internally and change its return type to `void` ?

Reply:
Refactored it.